yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -62,12 +63,25 @@ import { isMetricsLoggingRequested, setupMetricsLogger } from './logging.js';
 
 export { deployAndInitializeTokenAndBridgeContracts } from '../shared/cross_chain_test_harness.js';
 
-const { PXE_URL = '' } = process.env;
+const { PXE_URL = '', NOIR_RELEASE_DIR = 'noir-repo/target/release', TEMP_DIR = '/tmp' } = process.env;
 
 const getAztecUrl = () => {
   return PXE_URL;
 };
 
+// Determines if we have access to the acvm binary and a tmp folder for temp files
+const getACVMConfig = async (logger: DebugLogger) => {
+  try {
+    const expectedAcvmPath = path.resolve(`../../noir/${NOIR_RELEASE_DIR}`);
```

Comment:
I'd go with a `resolve(dirname(fileURLToPath(import.meta.url)), '../../../../noir/', NOIR_RELEASE_DIR)` (number of `../`s may be wrong), so the lookup doesn't depend on the working dir but this file's location.